client/src/containers/NychaPage.js

```diff
@@ -313,14 +313,12 @@ class NychaPageWithoutI18n extends Component {
                   location="nycha-page"
                   url={
                     usersInputAddress &&
-                    encodeURI(
-                      `https://whoownswhat.justfix.nyc${createRouteForAddressPage(
-                        usersInputAddress.boro,
-                        usersInputAddress.streetname,
-                        usersInputAddress.housenumber
-                      )}`
-                    ).replace(" ", "%20")
-                  } // Support for Android
+                    `${getSiteOrigin()}${createRouteForAddressPage({
+                      boro: usersInputAddress.boro,
+                      streetname: usersInputAddress.streetname,
+                      housenumber: usersInputAddress.housenumber,
+                    })}`
```

Comment:
Similarly, this would be a lot easier to read as `createRouteForAddressPage(usersInputAddress)`!